services/notifications/pkg/email/email.go

```diff
@@ -0,0 +1,40 @@
+package email
+
+import (
+	"bytes"
+	"embed"
+	"html/template"
+	"path/filepath"
+)
+
+var (
+	//go:embed templates
+	templatesFS embed.FS
+)
+
+// RenderEmailTemplate renders the email template for a new share
+func RenderEmailTemplate(templateName string, templateVariables map[string]string, emailTemplatePath string) (string, error) {
+	var err error
+	var tpl *template.Template
+	templateHasBeenFound := false
+	if emailTemplatePath != "" {
```

Comment:
Is this check really necessary? I would imagine that if the path is empty then you would receive a ErrNotExist and then `templateHasBeenFound` would still be false.